src/main/scala/uk/gov/hmrc/domain/nino.scala

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2015 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.domain
+
+import play.api.libs.json.{Reads, Writes}
+
+import scala.util.Try
+
+sealed trait ShortOrSuffixedNino extends TaxIdentifier with SimpleName {
+  def nino: String
+  final def value = nino
+  final val name = "nino"
+  override lazy val toString = nino
+
+  def formatted = value.grouped(2).mkString(" ")
+}
+
+object ShortOrSuffixedNino {
+  implicit val shortOrSuffixedNinoWrite: Writes[ShortOrSuffixedNino] = new SimpleObjectWrites[ShortOrSuffixedNino](_.value)
+  implicit val shortOrSuffixedNinoRead: Reads[ShortOrSuffixedNino] = new SimpleObjectReads[ShortOrSuffixedNino]("nino", ShortOrSuffixedNino.apply)
```

Comment:
I liked the idea of moving Json parsing related code from EmailAddress to PlayJsonFormats object, (https://github.com/hmrc/emailaddress/compare/release/0.2.0...v0.4.0), maybe the same changes could be applied to domain as well?